src/esp/gfx/Viewer.cpp

```diff
@@ -49,6 +49,9 @@ Viewer::Viewer(const Arguments& arguments)
       .addBooleanOption("enable-physics")
       .addOption("physicsConfig", "./data/default.phys_scene_config.json")
       .setHelp("physicsConfig", "physics scene config file")
+//      .addOption("lookAt", "0.0f,1.5f,0.0f,1.5f,1.0f,0.0f,1.0f,0.0f")
```

Comment:
@mosra : I am planning to offer an option to let the user provide the the projection parameters of the camera (eye, target, up). 
Question: if the input is a string as it shows here (0.0f,1.5f,0.0f,1.5f,1.0f,0.0f,1.0f,0.0f), do we have convenient functions in Corrade to parse it?

Reply:
Yes -- parse it as a Matrix4, tho it has to be delimited by spaces and without the `f` -- so 

```cpp
.addOption("lookAt", "0.0 1.5 0.0 1.5 1.0 0.0 1.0 0.0")
```

then you can get it conveniently as. Note the storage is column-major so depending on what you expect the users to do you might want to `transpose()` it after reading.

```cpp
Matrix4 lookAt = args.value<Matrix4>("lookAt");
```

By the way, some options are with `-` (`"enable-physics"`), some are camelCase -- can we stick to one style for good UX? So `"look-at"` in this case.